renovate.json

```diff
@@ -1,5 +1,9 @@
 {
-  "extends": [
-    "config:base"
-  ]
+  "extends": ["github>sourcegraph/renovate-config"],
+  "travis": {
+    "enabled": true
+  },
+  "node": {
+    "supportPolicy": ["lts"]
+  }
```

Comment:
We should set these things in https://github.com/sourcegraph/renovate-config if they are desirable

Reply:
^ not addressed

Reply:
Ah sorry, I misunderstood this comment.